bats/examples.bats

```diff
@@ -23,18 +32,18 @@ teardown_file() {
       }
     }'
   )
-
   exec_graphql 'cala-outbox-import-job-create' "$variables"
-
   name=$(graphql_output '.data.calaOutboxImportJobCreate.job.name')
-  [[ "$name" == "rust-example" ]] || exit 1;
+  error_msg=$(graphql_output '.errors[0].message')
+  [[ "$name" == "rust-example" || "$error_msg" =~ duplicate.*jobs_name_key ]] || exit 1;
```

Comment:
can't we add jitter to the name ( the "TEST_PREFIX" env var we discussed) instead of the cop-out via error.

Reply:
I was thinking this as well, but we didn't have a cancel job operation as yet and I wasn't sure what the behaviour would be if we had too many jobs running simultaneously.

We could also optimistically cancel/disable the job with this name and then run the create job to enable/re-enable it here. The `error_msg` check is more an interim solution.

Reply:
Yeah I think delete job and recreate would be good - should also fix the crash.